service/controller/app/app.go

```diff
@@ -113,6 +117,12 @@ func NewApp(config Config) (*App, error) {
 			Name: "app",
 		}
 
+		if !config.UniqueApp {
+			// Only watch app CRs in the current namespace. The label selector
+			// excludes the operator's own app CR which has the unique version.
+			c.Namespace = config.PodNamespace
```

Comment:
Sets the watch namespace for the app controller.

For the unique instance we continue to watch all namespaces.